packages/pie-toolbox/src/code/mask-markup/constructed-response.jsx

```diff
@@ -1,34 +1,75 @@
 import React from 'react';
-import Input from './components/input';
 import { withMask } from './with-mask';
+import EditableHtml from '../editable-html';
+import { withStyles } from '@material-ui/core/styles';
+import classnames from 'classnames';
+import { color } from '../../render-ui';
+
+const styles = (theme) => ({
+  editableHtmlCustom: {
+    display: 'inline-block',
+    verticalAlign: 'middle',
+    margin: '4px',
+    borderRadius: '4px',
+    border: `1px solid ${color.black()}`,
+  },
+  correct: {
+    border: `1px solid ${color.correct()}`,
+  },
+  incorrect: {
+    border: `1px solid ${color.incorrect()}`,
+  }
+});
 
 // eslint-disable-next-line react/display-name
-export default withMask('input', (props) => (node, data, onChange) => {
-  const dataset = node.data ? node.data.dataset || {} : {};
-  if (dataset.component === 'input') {
-    // eslint-disable-next-line react/prop-types
-    const { adjustedLimit, disabled, feedback, showCorrectAnswer, maxLength, spellCheck } = props;
+const MaskedInput = (props) => (node, data) => {
+  const { adjustedLimit, disabled, feedback, showCorrectAnswer, maxLength, spellCheck, classes, spanishInputEnabled, onChange } = props;
+  const dataset = node.data?.dataset || {};
 
-    // the first answer is the correct one
-    // eslint-disable-next-line react/prop-types
+  if (dataset.component === 'input') {
     const correctAnswer = ((props.choices && dataset && props.choices[dataset.id]) || [])[0];
     const finalValue = showCorrectAnswer ? correctAnswer && correctAnswer.label : data[dataset.id] || '';
     const width = maxLength && maxLength[dataset.id];
+    const languageCharactersProps = spanishInputEnabled ? [{ language: 'spanish' }] : [];
```

Comment:
We will need to implement a new property in the configuration that allows the customization of that response area, similar to what Adrian did here https://github.com/pie-framework/pie-elements/pull/2132/files#diff-6e29c8adb056d7113655814e0ead257da9e229a369dab2e5c170c38a0c202998R36 , except that for us, everything will have 'disabled: true' by default.